File: CS515/Assignment 2/src/Main.java
public class Main {
    public static void main(String[] args) {
        Account checkingAccount1 = new CheckingAccount("Account 1", 1, 0, 1.23, 100);
        Account savingsAccount2 = new SavingsAccount("Account 2", 2, 0, 1.23, true);
        Account account3 = new Account("Account 1", 1, 0, 1.23);

        Account[] accounts = { checkingAccount1, savingsAccount2, account3 };

        for (Account account : accounts) {
            account.print();
            System.out.println("-----------------------------------------------------------------------------------");
        }
    }
}

File: CS515/Assignment 2/C++/main.cpp
#include <iostream>
#include "Account.h"
#include "SavingsAccount.h"
#include "CheckingAccount.h"

int main() {
    Account account1("Account 1", 1, 1234, 1.23);
    SavingsAccount account2("Account 2", 2, 1234, 1.23, true);
    CheckingAccount account3("Account 3", 3, 1234, 1.23, 100);

    account1.print();
    std::cout << "------------------------------------------------ " << std::endl;
    account2.print();
    std::cout << "------------------------------------------------ " << std::endl;
    account3.print();

    return 0;
}
